Libs/MRML/Logic/vtkMRMLSliceLogic.h

```diff
@@ -71,10 +71,10 @@ class VTK_MRML_LOGIC_EXPORT vtkMRMLSliceLogic : public vtkMRMLAbstractLogic
 
   enum
   {
-    LayerNone = -1,
-    LayerBackground = 0,
-    LayerForeground = 1,
-    LayerLabel = 2,
+    LayerNone = vtkMRMLSliceCompositeNode::LayerNone,
+    LayerBackground = vtkMRMLSliceCompositeNode::LayerBackground,
+    LayerForeground = vtkMRMLSliceCompositeNode::LayerForeground,
+    LayerLabel = vtkMRMLSliceCompositeNode::LayerLabel,
     Layer_Last // must be last
```

Comment:
I purposefully did not replace use of the `vtkMRMLSliceLogic` enum with the `vtkMRMLSliceCompositeNode` one as we should warrant more discussion.

To ensure both are kept in sync, I will add a test to `MRMLLogic` checking that both enums have the same value of `Layer_Last`, and that the other values also match.